airbyte-db/lib/src/main/resources/configs_database/schema_dump.txt

```diff
@@ -1,8 +1,3 @@
-create type "source_type" as enum ('api', 'file', 'database', 'custom');
```

Comment:
We use the following piece of code to create the dump 
```
ExceptionWrappingDatabase(database).query(ctx -> ctx.meta().ddl().queryStream()
        .map(query -> query.toString() + ";")
        .filter(statement -> !statement.startsWith("create schema"))
        .collect(Collectors.joining("\n")));
```
and for some reason it doesnt acknowledge the enum types. I had added these manually but these keep getting deleted. AM trying to figure out how to stop it from deleting these

Reply:
the schema_dump file is just for representation purpose and isnt used to generate or initialize anything. added a comment to explain it